features/issue-13-indirect-import.feature

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2013-2015 BalaBit
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+
+Feature: Issue #13: indirect imports of foreign objects should not be checked
+
+  As a developer
+  In order to avoid conflicts with external libraries
+  I want typesafety to leave foreign objects alone
+
+  Scenario: autodecorator doesn't affect objects imported from external modules
+    Given that "external_lib.py" contains the following code:
+          """
+          def external_function(a: int) -> int:
```

Comment:
Then maybe we could write an annotaton that typesafety can understand, but should ignore anyways:

    should_be_ignored_by_typesafety = str
    
    def external_functio(a: should_be_ignored_by_typesafety) -> should_be_ignored_by_typesafety:
        return 1